src/web/Application.php

```diff
@@ -568,7 +568,7 @@ private function _processInstallRequest(Request $request)
         // Should they be accessing the installer?
         if (!$isInstalled) {
             if (!$isCpRequest) {
-                throw new ServiceUnavailableHttpException();
+                throw new ServiceUnavailableHttpException(Craft::t('app', 'Craft isn’t installed yet.'));
```

Comment:
This could result in a highly visible information disclosure vulnerability (revealing it’s a Craft site) if there’s an issue with the DB connection for an already live site.